Packs/ContentManagement/Scripts/MarketplacePackInstaller/MarketplacePackInstaller.py

```diff
@@ -211,6 +211,9 @@ def is_pack_already_installed(self, pack_data: Dict[str, str]) -> bool:
             return False
 
         try:
+            if pack_data['version'] in ['latest', '*']:
+                return parse(self.get_latest_version_for_pack(pack_data['id'])) == self.installed_packs[pack_data['id']]
+
             return parse(pack_data['version']) <= self.installed_packs[pack_data['id']]
```

Comment:
Why is the upper one with `id` == `id` and the lower one with `version` <= `id`? can you explain what each of them contains?